poolapi/apijson.go

```diff
@@ -0,0 +1,43 @@
+package poolapi
+
+import (
+	"encoding/json"
+)
+
+type Response struct {
```

Comment:
Client should be capable of mapping the response directly back to the request, so it knows what the expected data would be.